docs/source/RP3.rst

```diff
@@ -0,0 +1,210 @@
+*Credits: Thundercomm*
+
+After the OS image is successfully flashed, the following confirmation screen will appear.
+
+.. figure:: https://www.thundercomm.com/rubik-pi-3/en/assets/images/OS_image_flashing_completed-ff45bd2899b7b3e02bdb63f28a862f42.png
+    :align: center
+    :alt: Qualcomm launcher flash OS successful
+*Credits: Thundercomm*
+
+Configure the device
+-----------------
+
+Click Configure Login, the launcher will then prompt to setup UART connection.
+
+.. note::
+    If the MicroUSB COM port on the host system is currently in use by another tool (e.g., PuTTY, Tera Term), please close it before proceeding.
+
+Without disconnecting any cable from the device, connect your host computer to the device with a USB Micro-B cable at the [DEBUG] port
+
+.. figure:: /image/RubikPi/RubikPiUART.png
+    :scale: 30%
+    :alt: DEBUG port location
+
+Once the USB Micro-B cable is connected, a serial terminal will open. When asked, type in the default username and password:
+
+* Username: ubuntu
+* Password: ubuntu
+
+The terminal will ask you to reset the password immediately, type in the default password again when prompted, then set your password.
+
+The launcher will prompt you to configure the Wi-Fi once finished. Configure the Wi-Fi by entering the SSID and password, or connect the device to Ethernet.
+Once connected, the launcher will display the device IP address and the ``ssh`` command to connect to the device. Save this information.
+
+.. figure:: https://www.thundercomm.com/rubik-pi-3/en/assets/images/wi-fi_completed-6c17fb871952826a717480f620f4c29f.png
+    :align: center
+    :alt: Qualcomm launcher WiFi connect successful
+*Credits: Thundercomm*
+
+Click Finish Setup and verify that the setup process is complete.
+
+OpenCL Installation
+^^^^^^^^^^^^^^^^^^
+Now you can set up OpenCL on the Rubik Pi board.
+
+Connect to Rubik Pi
+------------------
+
+Open a terminal window on your host computer. Please use the ``ssh`` command displayed by the qualcomm launcher to connect to the Rubik Pi
+
+.. code-block:: bash
+    ssh ubuntu@RP3_IP_ADDRESS
+
+.. note::
+    When connecting for the first time, the terminal may prompt:
+    
+        The authenticity of host '192.168.0.*' can't be established.
+        ED25519 key fingerprint is SHA256:FINGERPRINT.
+        This key is not known by any other names. Are you sure you want to continue connecting (yes/no/[fingerprint])?
+
+    This is normal when conncting for the first time. However, if you see a big **WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!**,
+    and you have not caused the board's fingerprint to change, you may be a victim of a man-in-the-middle attack. (This should be *very* unlikely, but it’s good to be aware of them.)
+
+Package Installation
+-------------------
+OpenCL runtime should be pre-installed on the device, verify this by running:
+
+.. code-block:: bash
+    dpkg -L qcom-adreno1
+
+Verify that ``libOpenCL.so.1`` and ``libOpenCL_adreno.so.1`` are present.
```

Comment:
Make note to use grep, some students might forget about grep

Reply:
I added `grep "libOpenCL"`, the two libraries should be the only ones that show up before installing adreno dev packages